intro_to_rl.ipynb

```diff
@@ -0,0 +1,3225 @@
+{
```

Comment:
<p>This is a Haiku feature. Why not just use plain JAX random numbers? (so create a seed jax.random.PRNGKey(0) and then use <code>jax.random.split</code> inside <code>run_environment_loop</code>). Using any haiku features outside of a <code>hk.transform</code> is dangerous. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>